makefile

```diff
@@ -17,7 +17,7 @@ PLATFORM := $(shell uname | sed -e 's/_.*//')
 
 ifneq ($(MAKECMDGOALS),clean)
```

Comment:
perhaps remove this part completely

Reply:
Done.

Reply:
(Sorry, I hadn't meant to commit that commented-out part yet, originally.)